docs/explanations/audit-scanner.md

```diff
@@ -0,0 +1,235 @@
+
+See [here](../howtos/audit-scanner) how to enable Audit Scanner
+
+### Policy Reports
+
+When using the Kubewarden Audit Scanner, the results of the policy scans are
+stored in [PolicyReport](https://htmlpreview.github.io/?https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/docs/index.html) Custom Resource Definitions (CRDs).
+
+:::caution
+Note that the PolicyReport CRDs are under development in the `wg-policy`
+Kubernetes group.Therefore, this documentation can be out of date if a new
+version of the CRDs is released.
+
+Check the `wg-policy` group
+[repository](https://github.com/kubernetes-sigs/wg-policy-prototypes) out for
+more information about the CRDs.
+:::
+
+The audit scanner stores the audit results in the Policy Report Custom Resource
+Definitions (CRDs) created by the `wg-policy` Kubernetes group. These CRDs
+provide a structured way to store and manage the audit results. Each namespace
+scanned by the audit scanner will have a separate report associated with it.
+Cluster wide resources will have a separate report as well.
+
+To use the audit scanner, you need to manually install the Policy Report CRDs
+or use the version installed by `kubewarden-crds`.  These CRDs define the
+structure and schema for the audit reports generated by the scanner.
+
+The audit results generated by the scanner include various information, such as
+the policy that was evaluated, the resource being scanned, the result of the
+evaluation (pass, fail, or skip), and a timestamp indicating when the
+evaluation took place. Additionally, you can optionally define severity and
+category annotations for your policies in the `metadata.yml` file.
+
+You can also leverage the optional UI provided by the
+[policy-reporter](https://github.com/kyverno/policy-reporter) tool for
+monitoring and observability of the PolicyReport CRDs.  Furthermore, operators
+can access the reports via ordinary `kubectl` commands.
+
+
+Let's take a look at some example audit results generated by the audit scanner:
+
+### Cluster-Wide Audit Results example
+
+```yaml
+apiVersion: wgpolicyk8s.io/v1beta1
+kind: ClusterPolicyReport
+metadata:
+  creationTimestamp: "2023-07-10T19:25:40Z"
+  generation: 1
+  labels:
+    app.kubernetes.io/managed-by: kubewarden
+  ...
+results:
+- policy: cap-testing-cap-policy
+  ...
+  resourceSelector: {}
+  resources:
+  - apiVersion: v1
+    kind: Namespace
+    name: kube-system
+    ...
+  result: pass
+  rule: testing-cap-policy
+  source: kubewarden
+  timestamp:
+    nanos: 0
+    seconds: 1689017140
+- policy: cap-testing-cap-policy
+  ...
+  resourceSelector: {}
+  resources:
+  - apiVersion: v1
+    kind: Namespace
+    name: default
+    ...
+  result: pass
+  rule: testing-cap-policy
+  source: kubewarden
+  timestamp:
+    nanos: 0
+    seconds: 1689017140
+...
+summary:
+  error: 0
+  fail: 0
+  pass: 6
+  skip: 0
+  warn: 0
+```
+
+In the above example, the audit scanner has evaluated the
```

Comment:
I would prefer to have the explanations before the examples. So, "In the following example, this and that:" and then the Yaml.